tests/test_dynamodb/test_dynamodb_export_table.py

```diff
@@ -0,0 +1,357 @@
+
+
+@pytest.mark.aws_verified
+@dynamodb_aws_verified(create_table=False)
+def test_export_to_missing_s3_bucket(table_name=None):
+    client = boto3.client("dynamodb", region_name="us-east-1")
+
+    s3_bucket_name = f"inttest{uuid4()}"
+    s3_prefix = "prefix"
+    table_name = "test-table"
+
+    response = client.create_table(
+        TableName=table_name,
+        KeySchema=[{"AttributeName": "username", "KeyType": "HASH"}],
+        AttributeDefinitions=[{"AttributeName": "username", "AttributeType": "S"}],
+        ProvisionedThroughput={"ReadCapacityUnits": 5, "WriteCapacityUnits": 5},
+    )
+    table_arn = response["TableDescription"]["TableArn"]
+
+    client.update_continuous_backups(
+        TableName=table_name,
+        PointInTimeRecoverySpecification={"PointInTimeRecoveryEnabled": True},
+    )
+
+    export_description = client.export_table_to_point_in_time(
+        TableArn=table_arn,
+        ExportFormat="DYNAMO_JSON",
+        S3Bucket=s3_bucket_name,
+        S3Prefix=s3_prefix,
+    )["ExportDescription"]
+
+    export_details = wait_for_export(client, export_description)
+
+    assert export_details["ExportStatus"] == "FAILED"
+    assert export_details["FailureCode"] == "S3NoSuchBucket"
+    assert "The specified bucket does not exist" in export_details["FailureMessage"]
+
+
+@pytest.mark.aws_verified
+@dynamodb_aws_verified(create_table=False)
+def test_export_point_in_time_recovery_not_enabled(table_name=None):
+    client = boto3.client("dynamodb", region_name="us-east-2")
+    s3 = boto3.client("s3", region_name="us-east-1")
+
+    s3_bucket_name = f"inttest{uuid4()}"
+    s3_prefix = "prefix"
+    table_name = "moto_test_" + str(uuid4())[0:6]
+
+    s3.create_bucket(Bucket=s3_bucket_name)
+    response = client.create_table(
+        TableName=table_name,
+        KeySchema=[{"AttributeName": "username", "KeyType": "HASH"}],
+        AttributeDefinitions=[{"AttributeName": "username", "AttributeType": "S"}],
+        ProvisionedThroughput={"ReadCapacityUnits": 5, "WriteCapacityUnits": 5},
+    )
+
+    table_arn = response["TableDescription"]["TableArn"]
+
+    export_description = client.export_table_to_point_in_time(
+        TableArn=table_arn,
+        ExportFormat="DYNAMO_JSON",
+        S3Bucket=s3_bucket_name,
+        S3Prefix=s3_prefix,
+    )["ExportDescription"]
+
+    export_details = wait_for_export(client, export_description)
+
+    assert export_details["ExportStatus"] == "FAILED"
+    assert export_details["FailureCode"] == "PointInTimeRecoveryUnavailable"
+    assert (
+        "Point in time recovery not enabled for table"
+        in export_details["FailureMessage"]
+    )
+
+    client.delete_table(TableName=table_name)
+    s3.delete_bucket(Bucket=s3_bucket_name)
+
+
+@pytest.mark.aws_verified
+@dynamodb_aws_verified(create_table=False)
+@mock.patch.object(TableExport, "_backup_to_s3_file", mock.Mock(side_effect=Exception))
+def test_export_backup_to_s3_error(table_name=None):
```

Comment:
Patching doesn't work in server mode, so you'll need to add a guard clause.

See here for an example: https://github.com/getmoto/moto/blob/381f62f4cbd94e6e313b5a263ae469e1a24d54c8/tests/test_acm/test_acm.py#L681-L686

Reply:
fixed :)